src/__experimental__/components/numeral-date/numeral-date.component.js

```diff
@@ -8,15 +8,13 @@ import Textbox from '../textbox';
 const NumeralDate = ({
   dateFormat,
   defaultValue,
-  hasError,
-  hasInfo,
-  hasWarning,
+  error,
+  info,
+  warning,
   id,
-  inputIcon,
```

Comment:
We need to make sure we add `inputIcon` and `tooltipMessage` to the list of breaking changes.